release-common-lib/src/main/java/org/reactome/release/common/ReleaseStep.java

```diff
@@ -1,33 +1,57 @@
 package org.reactome.release.common;
 
-import java.io.FileNotFoundException;
-import java.io.IOException;
 import java.sql.SQLException;
 import java.util.Properties;
 
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.gk.persistence.MySQLAdaptor;
 
+/**
+ * This class provides the framework for a release step in Reactome's data-release-pipeline
+ * @author sshorser
+ */
 public abstract class ReleaseStep
 {
+
+	/**
+	 * General logger for this release step
+	 */
 	protected static final Logger logger = LogManager.getLogger();
-	
+
+	/**
+	 * Flag to determine if this step is being run live or as a test (false and true, respectively)
+	 */
 	protected boolean testMode;
-	
-	protected static MySQLAdaptor getMySQLAdaptorFromProperties(Properties props) throws IOException, FileNotFoundException, SQLException
+
+	/**
+	 * Parses a Properties object to obtain database connection information and return a database adaptor as a
+	 * MySQLAdaptor object.  Properties for db.host, db.user, db.password, db.name, and optionally db.port are
```

Comment:
`db.host` is also optional

Reply:
You're right - I'll update the JavaDoc to reflect that.

Reply:
Done!